repo/meta/index.json

```diff
@@ -22,6 +22,18 @@
       }
     },
     {
+      "currentVersion":"0.8.1-RC1",
+      "description":"A cluster-wide init and control system for services in cgroups or Docker containers.",
```

Comment:
Okay, I disagree with that one too. Here's a PR to change that: https://github.com/mesosphere/marathon/pull/1260